core/src/components/header/test/fade/header.e2e.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('header: fade', () => {
```

Comment:
This fix technically only applies to the `collapse` header. The example used in this test is confusing because it is a combination of the collapse+fade header. It might be better to have a separate `collapse` E2E test directory and fix this index.html file to not use the collapse header.